boa/src/exec.rs

```diff
@@ -507,6 +507,9 @@ impl Executor for Interpreter {
                     ValueData::Function(_) => "function",
                 }))
             }
+            ExprDef::CatchBlock(_, _) | ExprDef::TryCatch(_, _, _) => {
+                todo!("try/catch block execution")
```

Comment:
This should not panic, we should ideally check the result of the inner expression and pass the exception to the catch block.